jerry-core/ecma/builtin-objects/ecma-builtin-json.c

```diff
@@ -182,13 +187,17 @@ ecma_builtin_json_parse_string (ecma_json_token_t *token_p) /**< token argument
         }
         case LIT_CHAR_LOWERCASE_U:
         {
+          if ((end_p - current_p < ECMA_JSON_HEX_ESCAPE_SEQUENCE_LENGTH + 1)) {
+            return;
```

Comment:
what about <= ? Then the +1 is unnecessary.

Reply:
Done